### ISSUE TITLE
Russian translation issues

### ISSUE THREAD

**Issue by MaximP2014** [mozilla/legal-docs#1575]
Hi.
@pmac 
Please fix a few translation issues:

* https://github.com/mozilla/legal-docs/blob/master/ru/firefox_cloud_services_tos.md
> В этом разделе содержится краткий обзор перечисленных ниже условий. _Она_ поможет вам понять...

Correct: "**Он** поможет вам понять"

* https://github.com/mozilla/legal-docs/blob/master/ru/mozilla_privacy_policy.md
> Иногда мы раскрываем информацию с целью улучшения наших продуктов _лучше_  и _развитии_ открытого Интернета...

Correct: "**развития** открытого Интернета" & remove duplicate "лучше".

Good luck!

**Comment by peiying2** [comment]
Thank you @MaximP2014 for identifying the issues. I  need to have the Russian community to verify your change before I merge it. I will get back to you on this.

**Comment by MaximP2014** [comment]
Hi @peiying2 
Do you mean https://pontoon.mozilla.org/ru/ ?

**Comment by peiying2** [comment]
> Hi @peiying2
> Do you mean https://pontoon.mozilla.org/ru/ ?

Yes, do you know Alexander or Victor? I need to get confirmation from one of them. This is standard process.

**Comment by MaximP2014** [comment]
I've asked Valery Ledovskoy before. But he didn't found source texts. So I go here.

**Comment by BychekRU** [comment]
Hi! Thank you very much for reporting @MaximP2014
